processing/data_collection/gazette/spiders/rj_niteroi.py

```diff
@@ -0,0 +1,34 @@
+from gazette.items import Gazette
+import datetime as dt
+import scrapy
+
+
+class RjNiteroiSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '3303302'
+    name = 'rj_niteroi'
+    allowed_domains = ['niteroi.rj.gov.br']
+    start_urls = ['http://www.niteroi.rj.gov.br']
+    download_url = 'http://www.niteroi.rj.gov.br/downloads/do/{}/{}/{:02d}'
+    month_names = ['01_Jan', '02_Fev', '03_Mar', '04_Abr', '05_Mai',
+                   '06_Jun', '07_Jul', '08_Ago', '09_Set', '10_Out', '11_Nov', '12_Dez']
+
+    def parse(self, response):
+        """
+        @url http://www.niteroi.rj.gov.br
+        @returns items 1
+        @scrapes date file_urls municipality_id power scraped_at
+        """
+        start_date = dt.date(2015, 1, 1)
```

Comment:
I see no problem to get everything (not only from 2015-01-01). Items are dropped based on date in `BaseGazetteSpider` and there is nothing against getting everything available. According to Niteroi page, you can use 2003-07-01 as start date to retrieve the items:
"http://www.niteroi.rj.gov.br/do.html -  *Aviso: O diário oficial passou a ser fornecido a partir da data de Julho de 2003, o que inviabiliza qualquer D.O anterior."